src/ts/background/background.ts

```diff
@@ -25,14 +25,14 @@ browser.runtime.onMessage.addListener(async (message: DownloadMessage) => {
     if (message.type === DownloadType.single) {
         await downloadSingleImage(message);
     } else if (message.type === DownloadType.bulk) {
-        downloadBulk(message.imageURL, message.accountName);
+        downloadBulk(message);
     }
 });
 
 async function downloadSingleImage(message: DownloadMessage): Promise<void> {
     // Get the image id
     let imageName = getImageId(message.imageURL[0]);
-    imageName = `${message.accountName}_${imageName}`;
+    imageName = `${message.accountName}_${new Date(message.timestamp[0]*1000).toISOString().replace(/:/g, '-')}_${imageName}`;
```

Comment:
Perhaps we could use `new Date().toLocaleString("de-DE").replace(/(:|\/)/g, '.')` instead. 
I like the created output *"11.1.2021, 15.28.41"* a bit more like this

Reply:
> 
> 
> Perhaps we could use `new Date().toLocaleString("de-DE").replace(/(:|\/)/g, '.')` instead.
> I like the created output _"11.1.2021, 15.28.41"_ a bit more like this

I understand the aesthetic appeal, but the main reason I wanted to add this is for easy sorting of filenames chronologically, which means it has to be some sort of YEAR, MONTH, DAY, 24-HOUR, MINUTE, SECOND. Also sorry TypeScript/JavaScript is far from my native language of Python so I definitely broke a lot of rules unintentionally.

Reply:
OK this sounds reasonable. In this case don't change the timestamp.
Far from bad. And with the name public constructor I would have guessed Java and not Python ;)